build/helper/documentation_snippets.py

```diff
@@ -105,6 +114,29 @@
 '''
 
 
+def close_function_def_for_doc(functions, config):
+    # This is very specific to session based APIs. We look for a 'close' function and if we find it,
+    # We will copy that and modify it to be what we need for documentation
+    close_name = config['close_function']
+    if close_name in functions:
+        import copy
+        function_def = copy.deepcopy(functions[close_name])
+        if 'documentation' not in function_def:
+            function_def['documentation'] = {}
+        if 'description' not in function_def['documentation']:
+            function_def['documentation']['description'] = default_close_function_doc
+        if 'note' not in function_def['documentation']:
+            function_def['documentation']['note'] = []
+        if type(function_def['documentation']['note']) is not list:
+            function_def['documentation']['note'] = [function_def['documentation']['note']]
+        function_def['documentation']['note'].append(close_function_note)
+        function_def['python_name'] = 'close'
+    else:
+        assert False, "No '{}' function defined".format(close_name)
+
+    return function_def
+
+
 def initiate_function_def_for_doc(functions, config):
     # This is very specific to the IVI state model which not all drivers in nimi-python follow.
     # We look for a 'close' function and if we find it, We will copy that and modify it to be
```

Comment:
this is also outdated... there is no more "if we find it".

Reply:
Changed to something similar as for close.